### PR TITLE
Column Outline 

### PR DESCRIPTION
**Pull request by Devoodie**
Added the three classes that we confirmed we needed just to see if this is effective implementation, and also a knowledge check.

### CODE REVIEW

File: src/cookdevmax.py
Comment (Devoodie) [inline src/cookdevmax.py]: uhhh not this line of course...

**Review by CookieStudios5609 — CHANGES_REQUESTED** [review]
Make a new commit with this line removed before approval.
We should decide if we want to have our models in our base file or not. Remember, this file also has to hold all our routes, and our database logic too, if we don't separate them.
I vote for making a models subdirectory and importing them as needed.

**Pull request merged**
